### ISSUE TITLE
Audit all links and button text

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6475]
Check that all links and buttons contain "descriptive" text.  E.g. the user should be able to tell what the link or button does, without the context around it.

Example of link to be updated:

![Screenshot from 2019-08-17 07-21-14](https://user-images.githubusercontent.com/3323703/63207509-a0c40c00-c0bf-11e9-9b68-0077f7eda6ad.png)

The button here would be better as 'Add email'
